doc/base/classes.xml

```diff
@@ -12277,10 +12277,10 @@
 </class>
 <class name="Color" category="Built-In Types">
 	<brief_description>
-		Color in RGBA format.
+		Color in RGBA format with some support for ARGB format.
```

Comment:
I think it's bogus that the API uses ARGB instead of RGBA, but it's indeed the case currently so the docs are correct :)